creator-node/src/utils.js

```diff
@@ -216,8 +220,28 @@ async function rehydrateIpfsDirFromFsIfNecessary (req, dirHash) {
   }
 }
 
+async function fetchCIDFromOtherCreatorNodes (req, multihash, expectedStoragePath) {
+  const libs = req.app.get('audiusLibs')
+  const nodes = await libs.ethContracts.ServiceProviderFactoryClient.getServiceProviderList('creator-node')
```

Comment:
this should not be here. this list should get passed in somehow so we don't make this request for every missing file